dll/win32/shell32/shellmenu/CMenuSite.cpp

```diff
@@ -85,6 +85,9 @@ HRESULT STDMETHODCALLTYPE CMenuSite::AddBand(IUnknown * punk)
 
 HRESULT STDMETHODCALLTYPE CMenuSite::EnumBands(UINT uBand, DWORD* pdwBandID)
 {
+    if (uBand == 0xffffffff)
```

Comment:
```suggestion
    if (uBand == (UINT)-1)
```
or -1UL, or MAXUINT

Reply:
(note for reviewers: this implements the behaviour mentioned in https://learn.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-ibandsite-enumbands )